_posts/2021-04-09-springboot-autoconfigurations.md

```diff
@@ -0,0 +1,436 @@
+# How Spring Boot's AutoConfigurations Internally Work
+
+In this article, we will discuss and understand how Spring Boot's AutoConfigurations internally works
+
+## 1. Introduction
+Before we deep dive into Spring Boot's AutoConfiguration, **what really is Spring Boot?**
+
+*As per Spring Boot documentation:*
+> It takes an opinionated view of the Spring platform and third-party libraries so you can get started with minimum fuss.
```

Comment:
I don't think the reader needs an intro to Spring Boot when they land on this article. You can safely remove this section.